lua/exit.lua

```diff
@@ -45,9 +69,19 @@ Module.setup = function(config)
 end
 
 Module.prompt = function(prompt)
+  if not prompt or prompt == "" then
+    error("No prompt provided. Please specify a prompt to continue.")
+  end
+
   local cmd = Module.options.adapter.prompt(Module.options.model_name, prompt)
   local escaped = vim.fn.escape(cmd, '\\\"')
-  vim.api.nvim_feedkeys(escaped, "n", {})
+  if Module.options.adapter == 'ollama' then
```

Comment:
We should remove this ifs at some point, but I think it wasn't working because it should be:

```suggestion
  if Module.options.adapter.name == 'ollama' then
```